lib/plugins/aws/package/compile/events/stream/index.js

```diff
@@ -204,6 +204,10 @@ class AwsCompileStreamEvents {
               streamResource.Properties.MaximumBatchingWindowInSeconds = event.stream.batchWindow;
             }
 
+            if (event.stream.maximumRetryAttempts) {
```

Comment:
This will fail for `maximumRetryAttempts: 0`

Reply:
True, thanks for pointing. PR with a fix would be apprieciated